ckanext/datastore/tests/allowed_functions.txt

```diff
@@ -0,0 +1,2 @@
+upper
+"count"
```

Comment:
should that be "Counted" so its not in the list to verify that it does match and is case sensitive (2 tests, one passing with "Counted" and once failing with "COUNTED" etc.

Reply:
It has to be a real function name, or the test will fail for other reasons.

The unit test does verify that it works with lowercase, then fails with mixed case.

Reply:
all good then, might be good to squash this and provide a single commit to upstream ckan/ckan to integrate in the next patch ;)